node/blockchain/facade.py

```diff
@@ -143,9 +143,23 @@ def get_next_block_identifier(self) -> Optional[BlockIdentifier]:
 
         return HashableStringWrapper(last_block.body).make_hash()
 
+    @staticmethod
+    def create_account_state(account_number):
+        return AccountState(
+            _id=account_number,
+            balance=0,
+            account_lock=account_number,
+            node=None,
+        )
+
+    @staticmethod
+    def get_account_state(account_number) -> ORMAccountState:
+        return ORMAccountState.objects.get_or_none(_id=account_number
```

Comment:
why do not you use `update_or_create()`?

Reply:
Do you mean get_or_create()?